workflows/eduomics.nf

```diff
@@ -82,6 +82,11 @@ workflow EDUOMICS {
 
     ch_versions = ch_versions.mix(SIMULATE_RNASEQ_READS.out.versions)
 
+    // Set to true in test.config to run the test with a smaller dataset
+    if (params.istest) {
+        SIMULATE_RNASEQ_READS.out.simreads = SIMULATE_RNASEQ_READS.out.simreads.take(params.test_limit)
```

Comment:
we'll have to see whether an output channel is modifiable this way, or if we should instead create a new channel for this operation. I would suspect this second option is the safest in terms of execution.